common/app/views/fragments/analytics.scala.html

```diff
@@ -61,3 +62,27 @@
 </noscript>
 
 <img src="@Configuration.debug.beaconUrl/count/pv.gif" alt="" style="display : none ;" rel="nofollow"/>
+
+@if(NoBounceIndicator.isSwitchedOn) {
+    @* NOTE for removal - there is a corresponding entry in omniture.js *@
+    <script type="text/javascript">
```

Comment:
Inline script not ideal, but I want this separate from our Javascript system and switch will enforce deletion in the future.